tests/app/dao/test_annual_billing_dao.py

```diff
@@ -0,0 +1,35 @@
+from app.service.utils import get_current_financial_year_start_year
+from app.models import AnnualBilling
+from app.dao.annual_billing_dao import (
+    dao_create_or_update_annual_billing_for_year,
+    dao_get_free_sms_fragment_limit_for_year
+)
+
+
+def test_sample_service_has_free_sms_fragment_limit(notify_db_session, sample_service):
+
+    # when sample_service was created, it automatically create an entry in the annual_billing table
+    free_limit = dao_get_free_sms_fragment_limit_for_year(sample_service.id, get_current_financial_year_start_year())
+
+    assert free_limit.free_sms_fragment_limit == 250000
+    assert free_limit.financial_year_start == get_current_financial_year_start_year()
+    assert free_limit.service_id == sample_service.id
+
+
+def test_dao_update_free_sms_fragment_limit(notify_db_session, sample_service):
```

Comment:
Tests for unknown service_id?

Reply:
The endpoints are based on service id - would not get here if the service id is invalid. But I added a new test for this anyway. 

test_get_free_sms_fragment_limit_unknown_service_id